website/assets/styles/pages/docs/basic-documentation.less

```diff
@@ -498,6 +498,13 @@
 
       padding: 0px 40px;
 
+      .edit-button-container {
```

Comment:
looks like we're using `[purpose]` selectors everywhere else on this page rather than class selectors.  I think best to keep it consistent.